client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -608,6 +608,7 @@ export default {
 
     toggleRecommendationModal () {
       this.$refs.recommendationModal.toggle()
+      dplan.notify.notify('confirm', Translator.trans('recommendation.pasted'))
```

Comment:
I just realized that this is not the optimal place to call the notification, because the toggle method is also called when the modal is opened (so there's a notification when opening).